### ISSUE TITLE
Regex error

### ISSUE THREAD

**Issue by EntilZha** [orf/gping#2]
Getting what looks like a regex bug. Running on Mac OSX 10.11 with python 3.5

```python
$ gping google.com
Traceback (most recent call last):
  File "/usr/local/bin/gping", line 11, in <module>
    sys.exit(run())
  File "/usr/local/lib/python3.5/site-packages/gping/pinger.py", line 251, in run
    for ping in it(url):
  File "/usr/local/lib/python3.5/site-packages/gping/pinger.py", line 225, in _linux
    yield round(float(linux_re.search(line).group(1)))
AttributeError: 'NoneType' object has no attribute 'group'
```

Since there is apparently a failed match the match is None which is why it errors.

**Comment by KoesterD** [comment]
Can't reproduce this on Windows with Python 3.5.0. (Running "gping google.com")
May be OSX specific.

**Comment by jasonsperske** [comment]
When running ping on google I get this line:

    64 bytes from 216.58.192.46: icmp_seq=0 ttl=54 time=22.959 ms

The regex appears on line 18 on pinger.py is

    linux_re = re.compile(
    '.*?[+-]?\\d*\\.\\d+(?![-+0-9\\.]).*?[+-]?\\d*\\.\\d+(?![-+0-9\\.]).*?([+-]?\\d*\\.\\d+)(?![-+0-9\\.])',
    re.IGNORECASE | re.DOTALL)

**Comment by jasonsperske** [comment]
I haven't tested the portability of this but this regex works for me: `^.*\stime=([\.\d]*)\sms$`
(https://regex101.com/r/lB5nE1/1)

**Comment by KoesterD** [comment]
@jasonsperske There seems to be a fix for OSX in PR #7
